dps-web-experience-sample/Views/Shared/_DocsNav.cshtml

```diff
@@ -193,15 +191,15 @@
 
             <option value="~/en-us/docs/cordova-5-security/" id="cordova-5-security-option">Cordova 5 security</option>
 
-            <option value="~/">Configure your app</option>
+            <option value="~/en-us/docs/configure-app/" id="configure-app">Configure your app</option>
```

Comment:
Hi Norm! IDs have to be unique on each element of the page. These ones(small nav element) have "option" appended. So for "configure-app" document, id will be "configure-app-option" and so on. For the <li> elements above they are the same as the document id.